marketplace-builder/src/hooks.rs

```diff
@@ -82,7 +83,7 @@ impl BuilderHooks<SeqTypes> for EspressoReserveHooks {
             let bid_tx = match BidTxBody::new(
                 self.bid_key_pair.fee_account(),
                 self.bid_amount,
-                view_number + 3, // We submit a bid three views in advance.
+                view_number + 10, // We submit a bid ten views in advance.
```

Comment:
Did we end up deciding to submit 10 views in advance? Last time I checked it's 3 views: https://espresso.zulipchat.com/#narrow/stream/440749-Marketplace/topic/Async.20Standup/near/452603085, but maybe I missed a more recent discussion.

Reply:
No, that's my bad, I was testing something and forgot to remove this. 3 views is what we settled on, I reverted this